docs/sdkx_ios/sdk-configuration.mdx

```diff
@@ -337,6 +337,50 @@ Helpshift.showConversation(with:self, config:config)
 
 In your registration process, ask your user for his/her age. If the user's age is 13 or younger, set enableFullPrivacy to YES. This way, you comply with COPPA for your children users, but collect valuable user and device data for your other users.
 
+## initiateChatOnLoad {#initiateChatOnLoad}
+
+Upon setting `initiateChatOnLoad` to `YES`, when the conversation loads if the previous issue is resolved, a new conversation will be started, without end user clicking on the `New Conversation` button or going through the post resolution flows like feedback bots of previous issue.
+
+<Tabs>
+
+<TabItem value="Configuration details" label="Configuration details">
+
+|               |                                                   |
+| ------------- | ------------------------------------------------- |
+| Option:       | <code>initiateChatOnLoad</code>                   |
+| Values:       | <code>YES</code> / <code>NO</code>                |
+| Default:      | <code>NO</code>                                   |
+| Min SDK       | v10.0.0                                           |
+| Supported by: | <code>showConversation</code>                     |
```

Comment:
showFAQs will also support this right? For the scenario when conversation is opened from helpcenter.

Reply:
@jaydeep-helpshift we did not test with showFAQs, can we have a quick huddle over it

Reply:
the decision is to add it, if some customer demands for it